### ISSUE TITLE
关于结构

### ISSUE THREAD

**Issue by yehaizi1995** [Peterisfar/YOLOV3#3]
同学 你好 我想询问一下 你现在用的pytorch复现yolov3提高mAP时是通过改变网络结构来实现的吗?能说一下大体的改变吗

**Comment by Peterisfar** [comment]
Nope, just use some tricks of BoF and change the loss.

**Comment by yehaizi1995** [comment]
> Nope, just use some tricks of BoF and change the loss.

sorry,i don't know what it's the BoF,it's bag of word?

**Comment by Peterisfar** [comment]
a paper,  "Bag of Freebies for Training Object Detection Neural Networks"

**Comment by yehaizi1995** [comment]
> a paper, "Bag of Freebies for Training Object Detection Neural Networks"

thanks
